src/Optimization/optimization_problem_results.jl

```diff
@@ -58,6 +91,16 @@ function get_objective_value(res::OptimizationProblemResults, execution = 1)
     return res.optimizer_stats[execution, :objective_value]
 end
 
+function get_resolution(res::OptimizationProblemResults)
+    # Method return the first resolution between timestamps
+    # If single timestamp is used, it return nothing
+    diff_res = diff(get_timestamps(res))
+    if !isempty(diff_res)
+        return first(diff_res)
```

Comment:
Are we expecting that all the time increments should be the same? If so:
```suggestion
        return only(unique(diff_res))
```

If not, we should document clearly somewhere that `get_resolution` only gets the first increment, that seems like a surprising behavior

Reply:
I added the warning for the case of multiple resolutions.

We want to support being able to have multiple resolutions, but keep the structure mostly the same for PowerSimulations.